src/auth/auth.service.spec.ts

```diff
@@ -121,37 +79,108 @@ describe('AuthService', () => {
 
   describe('getTokensByUsername', () => {
     it('works', async () => {
+      jest
+        .spyOn(userRepo, 'findOne')
+        .mockResolvedValueOnce({ ...user, authTokens: [authToken] });
       const tokens = await service.getTokensByUsername(user.userName);
       expect(tokens).toHaveLength(1);
       expect(tokens).toEqual([authToken]);
     });
   });
 
   describe('getAuthToken', () => {
+    const token = 'testToken';
     it('works', async () => {
-      const token = 'testToken';
-      authToken.accessTokenHash = await service.hashPassword(token);
+      const accessTokenHash = await service.hashPassword(token);
+      jest.spyOn(authTokenRepo, 'findOne').mockResolvedValueOnce({
+        ...authToken,
+        user: user,
+        accessTokenHash: accessTokenHash,
+      });
       const authTokenFromCall = await service.getAuthTokenAndValidate(
         authToken.keyId,
         token,
       );
       expect(authTokenFromCall).toEqual({
         ...authToken,
         user: user,
+        accessTokenHash: accessTokenHash,
+      });
+    });
+    describe('fails', () => {
+      it(', because AuthToken could not be found', async () => {
```

Comment:
This ", because" description looks a bit weird in code.
To keep the intent, maybe use the following pattern instead?

```suggestion
    describe('fails: ', () => {
      it('AuthToken could not be found', async () => {
```

Reply:
done